src/openzaak/components/zaken/tests/test_zaak_closed.py

```diff
@@ -272,8 +272,14 @@ def test_klantcontacten(self):
         self.assertCreateBlocked(url, data)
 
     def test_rollen(self):
-        roltype = RolTypeFactory.create(zaaktype=self.zaaktype)
-        rol = RolFactory.create(zaak=self.zaak, roltype=roltype)
+        roltype = RolTypeFactory.create(
+            zaaktype=self.zaaktype, omschrijving_generiek=RolOmschrijving.initiator
+        )
+        rol = RolFactory.create(
+            zaak=self.zaak,
+            roltype=roltype,
+            omschrijving_generiek=RolOmschrijving.zaakcoordinator,
```

Comment:
this is not allowed to be different from the `roltype` value - maybe set it to `belanghebbende` instead? you can have multiple of those

Reply:
Thanks. I missed it. `omschrijving_generiek` values are replaces with `RolOmschrijving.behandelaar`